Bottom: 335d09f05387f1c700422e215bf3f2c9b7284a3c
Top:    7ff312579e0d5e58293bfa4bf19b2a1d6b6e0c86
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-13 22:27:01 +0000

Refresh of more-changes

---

diff --git a/src/polygon1.c b/src/polygon1.c
index a4378f8..3e7181d 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1071,9 +1071,6 @@ vertex_in_seg_rounded (const BoxType * b, void *cl)
   if (!line_point_inters (EDGE_BACKWARD_VERTEX (s->v)->point, EDGE_FORWARD_VERTEX (s->v)->point, i->v->point))
     return 0;
 
-  if (!line_point_inters (EDGE_BACKWARD_VERTEX (s->v)->point, EDGE_FORWARD_VERTEX (s->v)->point, i->v->point))
-    return 0;
-
   if (i->touch)			/* if checking touches one find and we're done */
     longjmp (*i->touch, TOUCHES);
